install/updateVersion.sh

```diff
@@ -216,15 +182,23 @@ EOF
 # Updating helm's values.yaml for the current versions
 #
 function update_helm_version() {
-   helm_values_file="$ROOT/install/kubernetes/helm/istio/values.yaml"
-   execute_sed "s|\(ca_hub: \).*|\1'${CA_HUB}'|g" $helm_values_file
-   execute_sed "s|\(ca_tag: \).*|\1'${CA_TAG}'|g" $helm_values_file
-   execute_sed "s|\(proxy_hub: \).*|\1'${PILOT_HUB}'|g" $helm_values_file
-   execute_sed "s|\(proxy_tag: \).*|\1'${PROXY_TAG}'|g" $helm_values_file
-   execute_sed "s|\(pilot_hub: \).*|\1'${PILOT_HUB}'|g" $helm_values_file
-   execute_sed "s|\(pilot_tag: \).*|\1'${PILOT_TAG}'|g" $helm_values_file
-   execute_sed "s|\(mixer_hub: \).*|\1'${MIXER_HUB}'|g" $helm_values_file
-   execute_sed "s|\(mixer_tag: \).*|\1'${MIXER_TAG}'|g" $helm_values_file
+  pushd $TEMP_DIR/templates/helm/istio
+  local HELM_FILE=$DEST_DIR/install/kubernetes/helm/istio/values.yaml
+
+  execute_sed "s|{CA_HUB}|${CA_HUB}|"       values.yaml.tmpl
+  execute_sed "s|{CA_TAG}|${CA_TAG}|"       values.yaml.tmpl
+  execute_sed "s|{PROXY_HUB}|${PILOT_HUB}|" values.yaml.tmpl
+  execute_sed "s|{PROXY_TAG}|${PROXY_TAG}|" values.yaml.tmpl
+  execute_sed "s|{PILOT_HUB}|${PILOT_HUB}|" values.yaml.tmpl
+  execute_sed "s|{PILOT_TAG}|${PILOT_TAG}|" values.yaml.tmpl
+  execute_sed "s|{MIXER_HUB}|${MIXER_HUB}|" values.yaml.tmpl
+  execute_sed "s|{MIXER_TAG}|${MIXER_TAG}|" values.yaml.tmpl
+
+  echo "# GENERATED FILE. Use with Kubernetes 1.7+" > $HELM_FILE
+  echo "# TO UPDATE, modify files in install/kubernetes/templates/helm/istio and run install/updateVersion.sh" >> $HELM_FILE
```

Comment:
>> will append.  In this case, don't we always want >?

@sbezverk re your comment about values.yaml not located in the correct place, this code appears to put the values.yaml in the correct location in the repository.